cav_msgs/msg/PredictedState.msg

```diff
@@ -0,0 +1,19 @@
+#
+# PredictedState.msg
+# 
+# Describes the predicted states of non-stationary external objects over one time step.
+
+# Header contains the frame rest of the fields will use
```

Comment:
Grammar here could be clearer
Header contains the timestamp and frame id which the remaining fields reference.